public/styles.css

```diff
@@ -0,0 +1,28 @@
+.wrapper {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+}
+
+.search {
+    margin: 10px;
+}
+
+.items {
+    display: flex;
+    align-items:flex-start;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+}
+
+.item {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    margin: 3px;
+}
+
+.img_user {
```

Comment:
just a suggestion: if image related to user then call css-class `user_img` instead of `img_user`. the fact that it is an image is important but the more important to whom this image belongs